gcconnex-kube.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: gcconnex-deployment
+  namespace: gctools-test
+spec:
+  replicas: 2 # tells deployment to run 2 pods matching the template
+  template: # create pods using pod definition in this template
+    metadata:
+      labels:
+        app: gcconnex
+    spec:
+      containers:
+      - name: gcconnex
+        image: localhost:5000/gctools-outilsgc/gcconnex:latest
+        ports:
+        - containerPort: 80
+        env:
+        - name: "DOCKER"
+          value: "1"
+        - name: "HOME"
+          value: "/var/www/html"
+        - name: "DBHOST"
+          value: "gcconnex-db"
```

Comment:
The database deployment will need a headless server declaration as well - the name to use here will be `servicename.gctools-test`